pkgs/development/python-modules/mpld3/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchPypi, python, buildPythonPackage
+, matplotlib, jinja2
+}:
+
+buildPythonPackage rec {
+  version = "0.3";
+  pname = "mpld3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4d455884a211bf99b37ecc760759435c7bb6a5955de47d8daf4967e301878ab7";
+  };
+
+  propagatedBuildInputs = [ matplotlib jinja2 ];
+  # Tests disabled as they need a working X display
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "D3 Viewer for Matplotlib";
```

Comment:
Please don't align these vertically, it's not a part of the nix style in the nixpkgs manual nor is it maintained.
Prospective formatters are being created without this as well.

Reply:
This was a reference from some other location. Now fixed to not be vertically aligned.